lines/client/config.py

```diff
@@ -2,7 +2,16 @@
 from enum import Enum
 
 
-class RundownSportId(Enum):
+class ExtendedEnum(Enum):
+    """ExtendedEnum delegation.."""
```

Comment:
Explain what this is doing.

Reply:
```python
    @classmethod
    def list(cls):
        """List ExtendedEnum for Enum."""
        return list(map(lambda c: c.value, cls))
```

The above provides an interface for `Exception` to provide all allowed values for a `RundownSportId`

Reply:
Write that in the docstring!